docs/supported-services.md

```diff
@@ -15,127 +15,128 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 
 # Supported services
 
```

Comment:
Why mess with reformatting this whole table?

I wouldn't say the result is more readable or easier to work with in a Markdown editor.

Reply:
Sorry! Using a new IDE, must have done it as pre-commit step or something - wasn't my intention to do so, I absolutely agree with you :) I'll revert that part

Reply:
Fixed :+1: